src/genomic/strand.rs

```diff
@@ -1,25 +1,25 @@
 use std::convert::TryFrom;
 use crate::SvartError;
 
-#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

Comment:
I am not sure we should promise `Copy` for `Strand`. Most of the time we do not need an owned `Strand` to do useful things (see my other review suggestions) and promising only `Clone` (and not `Copy`) is an easier requirement to satisfy in future.